pkgs/development/python-modules/pystac-client/default.nix

```
@@ -0,0 +1,84 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pythonOlder,
+
+  pystac,
+  pytest-benchmark,
+  pytest-console-scripts,
+  pytest-mock,
+  pytest-recording,
+  python-dateutil,
+  requests,
+  requests-mock,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pystac-client";
+  version = "0.8.3";
+  pyproject = true;
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "stac-utils";
+    repo = "pystac-client";
+    rev = "v${version}";
+    hash = "sha256-tzfpvNtj+KkKjA75En+OwxYQWGzxHLACLkzWT2j/ThU=";
+  };
+
+  build-system = [ setuptools ];
+
+  propagatedBuildInputs = [
+    pystac
+    python-dateutil
+    requests
+  ];
```

```suggestion
  ] ++ pystac.optional-dependencies.validation;
```
https://github.com/stac-utils/pystac-client/blob/b65b325d31019025faa17b32584c81967cbcc9e2/pyproject.toml#L29-L33

---

This doesn't work. 

```
       error: attribute 'optional-dependencies' missing

       at /home/imincik/Projects/nix/nixpkgs/code/pkgs/development/python-modules/pystac-client/default.nix:38:8:

           37|     requests
           38|   ] ++ pystac.optional-dependencies.validation;
             |        ^
           39|
```